.gitignore

```diff
@@ -9,3 +9,5 @@
 _build/
 build/
 dist/
+.venv
+.vscode
```

Comment:
```suggestion
```

Please do not add a .vscode folder. The .gitignore should be for universal things, such as build artifacts, but not preferences of IDEs. Use the `<base folder for asyncpraw>/.git/info/exclude` to exclude stuff locally without being pushed to remote..

Reply:
@PythonCoderAS  is it a convention followed in prawdev to avoid such things in .gitignore?

Reply:
Things like `.vscode` and `.idea` should be in the user's global `.gitignore`. The project level `.gitignore` is for artifacts made by the project itself. See https://github.com/praw-dev/praw/pull/1160 for more info.

Reply:
I am in favor for adding a `.venv` ignore. However, unless it is added to the main PRAW `.gitignore`, we'll leave it out of our `.gitignore` as well. I'd like to keep things standardized between the projects as much as possible.